extensions/wikia/ImageReview/ImageReviewImageStatus.php

```diff
@@ -11,76 +11,40 @@
 $wgHooks['ImagePageAfterImageLinks'][] = 'efImageReviewDisplayStatus';
 
 function efImageReviewDisplayStatus( ImagePage $imagePage, &$html ) {
-	global $wgCityId, $wgExternalDatawareDB, $wgUser, $wgImageReviewTestCommunities;
+	global $wgCityId, $wgExternalDatawareDB;
 
-	if ( !$wgUser->isAllowed( 'imagereviewstats' ) ) {
+	$context = $imagePage->getContext();
+
+	if ( !$context->getUser()->isAllowed( 'imagereviewstats' ) ) {
 		return true;
 	}
 
 	if ( !$imagePage->getTitle()->exists() ) {
 		return true;
 	}
 
-	$html .= Xml::element( 'h2', [], wfMessage( 'imagereview-imagepage-header' )->escaped() );
+	$html .= Xml::element( 'h2', [], $context->msg( 'imagereview-imagepage-header' )->escaped() );
 
 	$headers = [
-		wfMessage( 'imagereview-imagepage-table-header-reviewer' )->text(),
-		wfMessage( 'imagereview-imagepage-table-header-state' )->text(),
-		wfMessage( 'imagereview-imagepage-table-header-time' )->text(),
+		$context->msg( 'imagereview-imagepage-table-header-reviewer' )->text(),
+		$context->msg( 'imagereview-imagepage-table-header-state' )->text(),
+		$context->msg( 'imagereview-imagepage-table-header-time' )->text(),
 	];
 
 	$dbr = wfGetDB( DB_SLAVE, [], $wgExternalDatawareDB );
 
-	if ( in_array( $wgCityId, $wgImageReviewTestCommunities ) ) {
-		$reviews =
-			fetchReviewHistoryFromService( $wgCityId, $imagePage->getTitle() );
+	$reviews =
+		fetchReviewHistoryFromService( $wgCityId, $imagePage->getTitle() );
 
-		// TODO: temporary solution, remove it as soon as image review history is migrated to the new tool
-		// display history of review for images reviewed by old ImageReview tool
-		if ( empty( $reviews ) ) {
-			$reviews = fetchReviewHistory( $dbr, $wgCityId, $imagePage->getID() );
-		}
-	} else {
+	// TODO: temporary solution, remove it as soon as image review history is migrated to the new tool
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)